Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreRef.cpp

```diff
@@ -757,6 +757,14 @@ void WKWebsiteDataStoreRemoveAllFetchCaches(WKWebsiteDataStoreRef dataStoreRef,
     });
 }
 
+void WKWebsiteDataStoreRemoveNetworkCache(WKWebsiteDataStoreRef dataStoreRef, void* context, WKWebsiteDataStoreRemoveNetworkCacheCallback callback)
```

Comment:
I think this should have "DiskCache" in its name instead of "NetworkCache"
Even better would be to make a C API more like - (void)removeDataOfTypes:(NSSet<NSString *> *)dataTypes modifiedSince:(NSDate *)date completionHandler:(void (^)(void))completionHandler; where you can specify what types to remove rather than needing to make separate calls for each types.

Reply:
> I think this should have "DiskCache" in its name instead of "NetworkCache"

I put NetworkCache in this function, referring to the function below. I think it is better to use the same name.

https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreConfigurationRef.h#L42-L43
```
WK_EXPORT WKStringRef WKWebsiteDataStoreConfigurationCopyNetworkCacheDirectory(WKWebsiteDataStoreConfigurationRef configuration);
WK_EXPORT void WKWebsiteDataStoreConfigurationSetNetworkCacheDirectory(WKWebsiteDataStoreConfigurationRef configuration, WKStringRef directory);
```

Reply:
Hmm.  You're right.  This does follow the design of the WKWebsiteDataStoreRef API.  It would still be nice to have an API to clear all the types.  Much more future-proof and fewer roundtrips.